src/content/en/2020/seo.md

```diff
@@ -9,70 +8,72 @@ reviewers: [natedame, catalinred, dsottimano, dwsmart, en3r0, ibnesayeed, bazzad
 
-A robots.txt file must be plain text, encoded in UTF-8 and respond to requests with a 200 HTTP status code. A malformed robots.txt, a 4XX (client error) response, or more than 5 redirects are interpreted by search engine crawlers as a "full allow", meaning all content may be crawled. A 5XX (server error) response is understood as a "full disallow", meaning no content may be crawled. If the robots.txt is unreachable for more than 30 days, Google will use the last cached copy of it, as described in [their specifications](https://developers.google.com/search/reference/robots_txt#handling-http-result-codes).
+A `robots.txt` file must be plain text, encoded in UTF-8, and respond to requests with a 200 HTTP status code. A malformed `robots.txt`, a 4XX (client error) response, or more than five redirects are interpreted by search engine crawlers as a _full allow_, meaning all content may be crawled. A 5XX (server error) response is understood as a _full disallow_, meaning no content may be crawled. If the `robots.txt` is unreachable for more than 30 days, Google will use the last cached copy of it, as described in [their specifications](https://developers.google.com/search/reference/robots_txt#handling-http-result-codes).
 
-Overall, 80.05% of sites responded to robots.txt with a 2XX response. Of the 9,640,246 requests revolving with a 2XX response, 25.09% of these robots.txt files were not recognized as valid. This has slightly improved over 2019, when it was found that [72.16% of mobile sites had a valid robots.txt](../2019/seo#robotstxt).
+Overall, 80.05% of sites responded to `robots.txt` with a 2XX response. Of the 9,640,246 requests revolving with a 2XX response, 25.09% of these `robots.txt` files were not recognized as valid. This has slightly improved over 2019, when it was found that [72.16% of mobile sites had a valid `robots.txt`](../2019/seo#robotstxt).
 
-The data source for testing robots.txt validity, Lighthouse, introduced a [robots.txt audit](https://web.dev/robots-txt/) as part of the v6 update. This inclusion highlights that a successfully resolved request does not mean that the cornerstone file will be able to provide the necessary directives to web crawlers.
+The data source for testing `robots.txt` validity, Lighthouse, introduced a [robots.txt audit](https://web.dev/robots-txt/) as part of the v6 update. This inclusion highlights that a successfully resolved request does not mean that the cornerstone file will be able to provide the necessary directives to web crawlers.
 
+{# TODO(analysts, authors): Note that mobile and desktop can't be combined into "all devices" since they are overlapping datasets and most websites would be double-counted. When citing stats throughout the chapter, you need to specify which client you're referring to or include a disclaimer in the intro that stats are mobile unless specified otherwise. #}
```

Comment:
@aleyda @Tiggerito I haven't scanned the rest of the document but I'm afraid that this might be common throughout. Could you suggest changes or push commits directly to this branch to resolve any of these issues?

Reply:
@rviscomi  I had a scan through the markdown and only saw this case. I can fix this one. What's the process here, edit the latest draft, and remove the TODO lines, then commit?

I'm around for a few hours then out for the day.